react/features/large-video/components/LargeVideo.web.js

```diff
@@ -63,3 +72,24 @@ export default class LargeVideo extends Component<{}> {
         );
     }
 }
+
+
+/**
+ * Maps (parts of) the Redux state to the associated LargeVideo props.
+ *
+ * @param {Object} state - The Redux state.
+ * @private
+ * @returns {{
+ *     noAutoPlayVideo: boolean
+ * }}
+ */
+function _mapStateToProps(state) {
+    const testingConfig = state['features/base/config'];
+
+    return {
+        noAutoPlayVideo: testingConfig && testingConfig.noAutoPlayVideo
```

Comment:
Isn't this going to be actually config.noAutoPlayVideo and not config.testing.noAutoPlayVideo?